clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -175,7 +180,9 @@ public static void displayHelp(Options options) {
 
   /** Executes the appropriate command based on the command type. */
   private void executeCommand() {
-    if (entity.equals(CommandEntities.COLUMN)) {
+    if (command.equals(CommandActions.HELP)) {
```

Comment:
In the "set owner" PR #5293, you check the "--owner" flag; But here you use the command for such cross-entity action, I'm a little confused. Can it be consistent?

Reply:
That PR is not merged yet, this is based on code before that.

Reply:
The owner code has been merged